introducing-signal-store.md

```diff
@@ -0,0 +1,701 @@
+Welcome to **MiniRx Signal Store**, the new state management library from [MiniRx](https://mini-rx.io/).
+
+With the arrival of [Signals](https://angular.io/guide/signals) in Angular 16, new best practices are arising for state and events. In the future Signals and Observables will coexist in your application.
+
+Modern Angular needs modern state management which promotes **new Angular best practices** and **streamlines the usage of Signals and Observables**...
+
+MiniRx Signal Store is exactly that.
+
+### Modern state management with MiniRx Signal Store:
```

Comment:
AFAIU the biggest advantages of minirx signal store is it's kinda universal solution for both state management as well as the sync/async capabilities, right?

What I see in the section below (after reading half of the entire article) is that I'm kind of missing the _summary_, like, elevator pitch. The one sentence to convince me WHY should I choose this over anything else.

Is it a all-in-1 solution?

Maybe it is somewhere here, but it was not emphasized enough for me to spot it.